### PR TITLE
Fix the formatting of docstrings in load_earth_relief and load_earth_magnetic_anomaly

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

In the current documentation, the available `data_source` options are formatted in **bold**.
image

Since these options are strings that should be passed to `data_source` (e.g., `data_source="emga2"`), so they should be formatted like ``"emag2"`` instead.

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by yvonnefroehlich** [comment]
In this context, probably also the highlighting of the arguments for `terminator` of [`pygmt.Figure.solor`](https://www.pygmt.org/dev/api/generated/pygmt.Figure.solar.html) should be changed?

File: pygmt/datasets/earth_relief.py [hunk 1]
```diff
@@ -59,22 +59,20 @@ def load_earth_relief(
...
 
-        Available options:
-
-        - **igpp** : IGPP Global Earth Relief [Default option]. See
+        - ``"igpp"`` : IGPP Global Earth Relief [Default option]. See
           :gmt-datasets:`earth-relief.html`.
 
-        - **synbath** : IGPP Global Earth Relief dataset that uses
+        - ``"synbath"`` : IGPP Global Earth Relief dataset that uses
           stastical properties of young seafloor to provide more realistic
```
Comment (michaelgrund) [inline pygmt/datasets/earth_relief.py:68]:
```suggestion
          stastical properties of young seafloor to provide a more realistic
```

**Review by michaelgrund — APPROVED** [review]

File: pygmt/datasets/earth_magnetic_anomaly.py [hunk 2]
```diff
@@ -58,20 +58,18 @@ def load_earth_magnetic_anomaly(
         ``data_source="emag2_4km"``, which are ``"pixel"`` only.
 
     data_source : str
-        Select the source of the magnetic anomaly data.
+        Select the source of the magnetic anomaly data. Available options are:
 
-        Available options:
-
-        - **emag2** : EMAG2 Global Earth Magnetic Anomaly Model [Default
+        - ``"emag2"`` : EMAG2 Global Earth Magnetic Anomaly Model [Default
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_magnetic_anomaly.py:63]: Is there an intention behind setting a white space before the `:`?
```suggestion
        - ``"emag2"``: EMAG2 Global Earth Magnetic Anomaly Model [Default
```

File: pygmt/datasets/earth_magnetic_anomaly.py [hunk 3]
```diff
@@ -58,20 +58,18 @@ def load_earth_magnetic_anomaly(
...
           option]. Only includes data observed at sea level over
           oceanic regions. See :gmt-datasets:`earth-mag.html`.
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_magnetic_anomaly.py:65]:
```suggestion
          option]. It only includes data observed at sea level over
          oceanic regions. See :gmt-datasets:`earth-mag.html`.
```

File: pygmt/datasets/earth_magnetic_anomaly.py [hunk 4]
```diff
@@ -58,20 +58,18 @@ def load_earth_magnetic_anomaly(
...
 
-        - **emag2_4km** : Use a version of EMAG2 where all observations
+        - ``"emag2_4km"`` : Use a version of EMAG2 where all observations
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_magnetic_anomaly.py:67]:
```suggestion
        - ``"emag2_4km"``: Use a version of EMAG2 where all observations
```

File: pygmt/datasets/earth_magnetic_anomaly.py [hunk 5]
```diff
@@ -58,20 +58,18 @@ def load_earth_magnetic_anomaly(
...
           are relative to an altitude of 4 km above the geoid and include
           data over land.
 
-        - **wdmam** : World Digital Magnetic Anomaly Map (WDMAM).
-          See :gmt-datasets:`earth-wdmam.html`
+        - ``"wdmam"`` : World Digital Magnetic Anomaly Map (WDMAM).
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_magnetic_anomaly.py:71]:
```suggestion
        - ``"wdmam"``: World Digital Magnetic Anomaly Map (WDMAM).
```

File: pygmt/datasets/earth_relief.py [hunk 6]
```diff
@@ -59,22 +59,20 @@ def load_earth_relief(
         for all resolutions except ``"15s"`` which is ``"pixel"`` only.
 
     data_source : str
-        Select the source for the Earth relief data.
+        Select the source for the Earth relief data. Available options are:
 
-        Available options:
-
-        - **igpp** : IGPP Global Earth Relief [Default option]. See
+        - ``"igpp"`` : IGPP Global Earth Relief [Default option]. See
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_relief.py:64]:
```suggestion
        - ``"igpp"``: IGPP Global Earth Relief [Default option]. See
```

File: pygmt/datasets/earth_relief.py [hunk 7]
```diff
@@ -59,22 +59,20 @@ def load_earth_relief(
...
           :gmt-datasets:`earth-relief.html`.
 
-        - **synbath** : IGPP Global Earth Relief dataset that uses
+        - ``"synbath"`` : IGPP Global Earth Relief dataset that uses
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_relief.py:67]:
```suggestion
        - ``"synbath"``: IGPP Global Earth Relief dataset that uses
```

File: pygmt/datasets/earth_relief.py [hunk 8]
```diff
@@ -59,22 +59,20 @@ def load_earth_relief(
...
           stastical properties of young seafloor to provide more realistic
           relief of young areas with small seamounts.
 
-        - **gebco** : GEBCO Global Earth Relief with only observed relief and
+        - ``"gebco"`` : GEBCO Global Earth Relief with only observed relief and
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_relief.py:71]:
```suggestion
        - ``"gebco"``: GEBCO Global Earth Relief with only observed relief and
```

File: pygmt/datasets/earth_relief.py [hunk 9]
```diff
@@ -59,22 +59,20 @@ def load_earth_relief(
...
           inferred relief via altimetric gravity. See
           :gmt-datasets:`earth-gebco.html`.
 
-        - **gebcosi** : GEBCO Global Earth Relief that gives sub-ice (si)
+        - ``"gebcosi"`` : GEBCO Global Earth Relief that gives sub-ice (si)
```
Comment (yvonnefroehlich) [inline pygmt/datasets/earth_relief.py:75]:
```suggestion
        - ``"gebcosi"``: GEBCO Global Earth Relief that gives sub-ice (si)
```

**Comment by yvonnefroehlich** [comment]
Hm. I am wondering, whether the unit "Tesla" really has a plural form:
Lines 76-80 in https://github.com/GenericMappingTools/pygmt/blob/main/pygmt/datasets/earth_magnetic_anomaly.py:
```python
    Returns
    -------
    grid : :class:`xarray.DataArray`
        The Earth magnetic anomaly grid. Coordinates are latitude and
        longitude in degrees. Units are in nano Teslas (nT).
```

**Comment by michaelgrund** [comment]
> Hm. I am wondering, whether the unit "Tesla" really has a plural form, please see Lines 76-80 in https://github.com/GenericMappingTools/pygmt/blob/main/pygmt/datasets/earth_magnetic_anomaly.py:
> 
> ```python
>     Returns
>     -------
>     grid : :class:`xarray.DataArray`
>         The Earth magnetic anomaly grid. Coordinates are latitude and
>         longitude in degrees. Units are in nano Teslas (nT).
> ```

Yes should be nano Tesla since the plural of Tesla is also Tesla.

**Comment by seisman** [comment]
> > Hm. I am wondering, whether the unit "Tesla" really has a plural form, please see Lines 76-80 in [`main`/pygmt/datasets/earth_magnetic_anomaly.py](https://github.com/GenericMappingTools/pygmt/blob/main/pygmt/datasets/earth_magnetic_anomaly.py?rgh-link-date=2023-01-20T12%3A45%3A37Z):
> > ```python
> >     Returns
> >     -------
> >     grid : :class:`xarray.DataArray`
> >         The Earth magnetic anomaly grid. Coordinates are latitude and
> >         longitude in degrees. Units are in nano Teslas (nT).
> > ```
> 
> Yes should be nano Tesla since the plural of Tesla is also Tesla.

Done in 4bc4de4a8f2e41b5d812e48f998637c682d84ebd.

**Review by michaelgrund — APPROVED** [review]
Looks fine!

**Review by yvonnefroehlich — COMMENTED** [review]
Except for `pygmt.datasets.load_earth_relief` (line 26):
load_earth_relief_url

we highlight the directory to which the grids are downloaded like this, e.g., `pygmt.datasets.load_earth_magnetic_anomaly`:
load_earth_magnetic_anomaly_url

I think we should do this consistently in the same way across all functions.

**Comment by seisman** [comment]
> Except for `pygmt.datasets.load_earth_relief` (line 26): load_earth_relief_url
> 
> we highlight the directory to which the grids are downloaded like this, e.g., `pygmt.datasets.load_earth_magnetic_anomaly`: load_earth_magnetic_anomaly_url
> 
> I think we should do this consistently in the same way across all functions.

Done in be9b00240.

**Review by yvonnefroehlich — APPROVED** [review]
Looks good!

**Pull request merged**